### PR TITLE
FRW-565 Updated Integrator in order to have possibility to change the manifest source

### PR DESCRIPTION
**Pull request by yaroslav-spryker**
Ticket: https://spryker.atlassian.net/browse/FRW-565

### CODE REVIEW

File: src/Console/ModuleInstallerConsole.php [hunk 1]
```diff
@@ -31,6 +32,36 @@ class ModuleInstallerConsole extends Command
      */
     protected const ARGUMENT_MODULE_NAMES = 'module-names';
 
+    /**
+     * @var string
+     */
+    protected const ARGUMENT_MODULE_NAMES_DESCRIPTION = 'Name of modules which should be build, separated by `,`';
```
Comment (dereuromark) [inline src/Console/ModuleInstallerConsole.php:38]: typo built

**Review by gechetspr — COMMENTED** [review]
Looks good. Add tests and make sure that regular flow (without source) works too

**Review by gechetspr — APPROVED** [review]

**Review by olhalivitchuk — APPROVED** [review]

**Comment by DmytroKlymanSpryker** [comment]
@yaroslav-spryker , should I release and ship this PR?

**Comment by yaroslav-spryker** [comment]
> @yaroslav-spryker , should I release and ship this PR?

@DmytroKlymanSpryker This one also needs a QA, I`ll let you know when we can merge

**Review by gechetspr — APPROVED** [review]

**Comment by gechetspr** [comment]
@DmytroKlymanSpryker we are done with QA so please release it if you can :)

**Pull request merged**